app/models/dog.rb

```diff
@@ -0,0 +1,11 @@
+class Dog < ActiveRecord::Base
+  belongs_to :user
+
+  has_attached_file :img,
+    styles: {medium: '300x300>'},
+    storage: :s3,
+    bucket: ENV['BUCKET_NAME'] 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.